docs/meshstack.metadata-tags.md

```diff
@@ -22,168 +25,42 @@ such metadata include:
-      "$id": "/properties/dataClassification",
-      "type": "string",
-      "title": "Data Classification (by admin)",
-      "description": "As recorded during the approval process",
-      "restricted": true,
-      "oneOf": [
-        {
-          "description": "Internal",
-          "enum": ["internal"]
-        },
-        {
-          "description": "Confidential",
-          "enum": ["confidential"]
-        },
-        {
-          "description": "Top Secret",
-          "enum": ["top-secret"]
-        }
-      ],
-      "default": "internal"
-    },
-    "environment": {
-      "$id": "/properties/environment",
-      "type": "array",
-      "title": "Environment",
-      "description": "The environment(s) the customer uses.",
-      "minItems": 1,
-      "items": {
-        "type": "string",
-        "oneOf": [
-          {
-            "description": "Development",
-            "enum": [
-              "dev"
-            ]
-          },
-          {
-            "description": "Test",
-            "enum": [
-              "test"
-            ]
-          },
-          {
-            "description": "QA",
-            "enum": [
-              "qa"
-            ]
-          },
-          {
-            "description": "Production",
-            "enum": [
-              "prod"
-            ]
-          }
-        ]
-      },
-      "widget": "multiselect",
-      "default": ["dev"],
-      "restricted": true
-    }
-  },
-  "required": [
-    "costCenter"
-  ]
-}
-```
+## Viewing and defining available tags
 
-### Configuration Model
+> Note: doing the actions below will require the Partner Admin role.
 
-Operators can configure tag schemas in the [meshStack configuration model](meshstack.configuration.md) under `meshfed.web` as follows:
+To get started with managing tags in meshstack, navigate to the Tags page which is available in the Administration area:
 
-```dhall
-{
-  customer-tag-schema = Some "./my-customer-tag-schema.json"
-, project-tag-schema = Some "./my-project-tag-schema.json"
-, landing-zone-tag-schema = Some "./my-landing-zone-tag-schema.json"
-, payment-method-tag-schema = Some "./payment-method-tag-schema.json"
-}
-```
+![Tag List](assets/metadata_tags/admin_area_tags_list.png)
 
-The schemas need to be available in the configuration repository.
+This page will show you all tags that are currently active on supported meshObjects. If you would like to know more details about a tag, or create, edit, or delete a tag, you can do so by clicking on the large arrow ▶️ on the right for a meshObject.
+This next page will show you more details about a tag such as the title, description or data type of a tag.
 
-### Tags on Payment Methods
+### Defining a new tag
 
-[Payment Methods](meshcloud.payment-methods.md) can also provide tag information. For example,
-the default "cost center" payment method provides the `costCenter` tag.
+You can define a new tag by clicking '+ Create' at the top right in the page mentioned above.
```

Comment:
I find this misleading as i connected "the page mentioned above" with the screenshot, but the create button is only available on one of the sub-screens.